src/Annotations/Serializer/CommonMark/Block/Parser/MathMLParser.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace eLife\Annotations\Serializer\CommonMark\Block\Parser;
+
+use eLife\Annotations\Serializer\CommonMark\Block\Element\MathML;
+use League\CommonMark\Block\Parser\AbstractBlockParser;
+use League\CommonMark\ContextInterface;
+use League\CommonMark\Cursor;
+
+class MathMLParser extends AbstractBlockParser
```

Comment:
would be helpful to document in the README where the architecture of these parsers come from, whether it is documentation or helpful Stack Overflow questions. A couple of links which will be very valuable when someone needs to make changes down the road